apps/complaints/acceptance/features/confirmation.js

```diff
@@ -11,14 +11,16 @@ Before((
   I.visitPage(confirmationPage, steps);
 });
 
-Scenario('The correct fields elements are on the page', function *(
+Scenario('The correct page is served', function *(
   I,
   confirmationPage
 ) {
-  yield I.setSessionData(steps.name, {
-    'email-address': 'test@test.com'
-  });
+  yield I.amOnPage('/' + confirmationPage.url);
+});
+
+Scenario('The first page is served when the page is refreshed', function *(
+  I
+) {
   yield I.refreshPage();
-  I.seeElement(confirmationPage.sentHeader);
-  I.see(confirmationPage.content.email);
+  I.amOnPage('/');
```

Comment:
`amOnPage` actually navigates to the given URL. If you want to check you are on the correct page you should use `I.seeInCurrentUrl`